toptal/picasso
# feat(icon): add gift icon

### Description

Add Gift icon

### How to test

- `yarn start`
- [ensure the Link icon looks good](http://localhost:9001/?path=/story/components-folder--icons)

or check the [temploy](https://picasso.toptal.net/add-gift-icon/?path=/story/components-folder--icons)

### Screenshots

![image](https://user-images.githubusercontent.com/1213542/89065545-465ca380-d342-11ea-9cd8-e3be4c023d42.png)

### Review

- [n/a] Annotate all `props` in component with documentation
- [n/a] Create `examples` for component
